recipe/meta.yaml

```
@@ -20,10 +22,11 @@ source:
     - patches/remove_nab2c_and_friends.patch  # [arm64]
 
 build:
-  number: 6
+  number: {{ build }}
   skip: True  # [win or py2k or ppc64le or aarch64]
```

```suggestion
  skip: True  # [win or py2k or ppc64le or aarch64 or py!=39]
```

Do note the explosion of items in the job matrix. Might be preferable to keep it to a single python version while iterating. Add this and rerender if you agree.

---

+1 While this can be done locally, at the end of the day it will require some ping-pong with the CI, so turning off builds you don't need and working on one at a time is how I like to do it